parcels/field.py

```diff
@@ -628,8 +629,8 @@ def interpolator2D(self, ti, z, y, x):
             raise RuntimeError(self.interp_method+" is not implemented for 2D grids")
 
     def interpolator3D(self, ti, z, y, x, time):
-        xi = int(self.grid.xdim / 2)
-        yi = int(self.grid.ydim / 2)
+        xi = int(self.grid.xdim / 2) - 1
+        yi = int(self.grid.ydim / 2) - 1
```

Comment:
Why this change? Just for future reference, would be good to mention why we need to subtract 1 here, and then add it later in this function

Reply:
We don't add it later in the function.
This is the initial xi, yi guess before looking really in which cell we are (in scipy mode, xi and yi are not attached to the particle)

For the smaller possible element, xdim=ydim=2
So you can see that initial xi and yi are 1, which does not make sense since it ranges from 0 to 1. This is why there is this -1